api/lib/utils/scalingEngineUtils.js

```diff
@@ -14,6 +14,18 @@ module.exports = function(scalingEngineSettings) {
       timeout: 10000,
     }
     if (scalingEngineSettings.tls) {
+      if(!fs.existsSync(scalingEngineSettings.tls.keyFile)){
+        logger.error("Invalid TLS key path: " + scalingEngineSettings.tls.keyFile);
+        throw new Error("Invalid TLS key path: " + scalingEngineSettings.tls.keyFile);
+      }
+      if(!fs.existsSync(scalingEngineSettings.tls.certFile)){
+        logger.error("Invalid TLS certificate path: " + scalingEngineSettings.tls.certFile);
+        throw new Error("Invalid TLS certificate path: " + scalingEngineSettings.tls.certFile);
+      }
+      if(!fs.existsSync(scalingEngineSettings.tls.caCertFile)){
+        logger.error("Invalid TLS ca certificate path: " + scalingEngineSettings.tls.caCertFile);
+        throw new Error("Invalid TLS ca certificate path: " + scalingEngineSettings.tls.caCertFile);
+      }      
```

Comment:
same as above

Reply:
I removed this from this PR. will resubmit later